src/app/auth/store/auth.effects.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { of } from 'rxjs';
+import { map, mergeMap, catchError } from 'rxjs/operators';
+import { FailedRequest, SuccessfulRequest } from 'src/app/app-model';
+
+import { AuthService } from '../services/auth.service';
+import { AuthActions } from './auth.actions';
+
+@Injectable()
+export class AuthEffects {
```

Comment:
The same as courses and authors, please check all **unused** code.

Reply:
checked